src/index.html

```diff
@@ -6,9 +6,46 @@
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>Moyo header</title>
+  <link href="https://fonts.googleapis.com/css?family=Roboto:500&display=swap" rel="stylesheet">
   <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-  <h1>Moyo header</h1>
+
+  <header class="header">
+
+    <a href="/" class="header__logo-link">
+      <img src="/images/logo.png" alt="logo" class="header__logo">
```

Comment:
```suggestion
      <img src="./images/logo.png" alt="logo" class="header__logo">
```
картинка не підгрузилась тому що нема крапочки)